dspace-api/src/main/java/org/dspace/scripts/service/ProcessService.java

```diff
@@ -32,11 +33,14 @@ public interface ProcessService {
      * @param ePerson       The ePerson for which this process will be created on
      * @param scriptName    The script name to be used for the process
      * @param parameters    The parameters to be used for the process
+     * @param specialGroups Allows to set special groups, associated with application context when process is created,
+     *                      other than the ones derived from the eperson membership.
      * @return The created process
      * @throws SQLException If something goes wrong
      */
     public Process create(Context context, EPerson ePerson, String scriptName,
-                          List<DSpaceCommandLineParameter> parameters) throws SQLException;
+                          List<DSpaceCommandLineParameter> parameters,
+                          final List<Group> specialGroups) throws SQLException;
```

Comment:
as specialGroups are now a Set see https://github.com/DSpace/DSpace/blob/main/dspace-api/src/main/java/org/dspace/core/Context.java#L95 we should use set here as well